src/locales/ko/battle.json

```
@@ -32,6 +32,7 @@
   "attackMissed": "{{pokemonNameWithAffix}}에게는\n맞지 않았다!",
   "attackHitsCount": "{{count}}번 맞았다!",
   "rewardGain": "{{modifierName}}[[를]] 받았다!",
+  "rewardGainCount": "{{modifierName}}[[를]]\n{{count}}개 손에 넣었다!",
```

```suggestion
  "rewardGainCount": "$t(modifierName, {\"count\": {{count}} })[[를]]\n{{count}}개 손에 넣었다!",
```

For possible plural handling

DISCLAIMER: Does `ko` need this kind of plural handling?